platform/resource-bundle-locator/pom.xml

```diff
@@ -88,7 +88,7 @@
                     <limit implementation="org.codice.jacoco.LenientLimit">
```

Comment:
✏️ If this is meant to be skipped because it is a pure interface than do
```
      <plugin>
                <groupId>org.jacoco</groupId>
                <artifactId>jacoco-maven-plugin</artifactId>
                <configuration>
                    <skip>true</skip>
                </configuration>
            </plugin>
```

Reply:
This was a mistake. I meant to change the coverage to 0.70 but instead did 0.00